python/dgl/graphbolt/feature_store.py

```diff
@@ -0,0 +1,166 @@
+"""Feature store for GraphBolt."""
+import torch
+
+
+class FeatureStore:
+    r"""Base class for feature store."""
+
+    def __init__(self):
+        pass
+
+    def read_feature(self, key: str, ids: torch.Tensor = None):
```

Comment:
redundant _feature in each method, just read / add / update or maybe just get / set? not sure why we need add.

Reply:
I'm not sure whether we need to `add_feature` (which also adds a new key) in the future. If not, all the keys of the FeatureStorage are initialized in the constructor. Since it is not clear, I will remove this interface now.

Comparing `set/get` and `read/update`, I prefer the latter because if we use some ids to access a feature, we only read or update a part of the feature, instead of setting or getting the whole feature.

Reply:
What's the API name for tensor?

Reply:
Sounds good.

Reply:
read_feature -> read is enough, FeatureStore's read method implies read_feature.

Reply:
Right!